pkg/txm/clientwrappers/dualbroadcast/flashbots_client.go

```diff
@@ -20,25 +20,25 @@ import (
 	"github.com/smartcontractkit/chainlink-evm/pkg/txm/types"
 )
 
-type DualBroadcastClient struct {
+type FlashbotsClient struct {
```

Comment:
While we're renaming, can we make it unexported to limit the API surface?
```suggestion
type flashbotsClient struct {
```

Reply:
Exporting it makes our lives easier during testing because we can import it locally and run our tests outside the Core node.

Reply:
Makes sense 👍